<center>
<i>Welcome to the <em>Population Genomics</em> Laboratory* of the <a href="http://www.igb.cnr.it/">Institute of Genetics and Biophysics</a> of the  <a href="https://www.cnr.it/en">National Research Council</a>. We are interested in understanding causes and consequences of genetic diversity and how natural selection in humans affects loci related to diseases</i>
</center>

<blockquote>
  <p>“<em>Fascinating</em>” <em>(Lieutenant Spock)</em></p>
</blockquote>

<p><img src="../images/loghi_cnr_igb.png" alt="logoCNR" /></p>

<h1 id="research"><em>Research</em></h1>

<h2 id="genomics-of-early-embryonic-development">Genomics of early embryonic development</h2>

<p><i> Hownatural selection acts on early human development</i></p>

<p>We investigate adverse outcomes of embryonic development like recurrent pregnancy loss and preeclampsia to identify the genetic factors that influence reproductive outcomes and pregnancy complications. This knowledge furthers our understanding of human evolution and informs efforts to improve pregnancy outcomes.</p>

<p><img src="images/embryo.png" alt="embryo" /></p>

<p>In <b>CABERNET</b> we aim to determine the extent of <em>chromosomal mosaicism between embryonic and extraembryonic tissues</em> using single cell DNA sequencing.
In <b>DELIVER / NEONATE</b> we want to identify genetic factors contributing to <em>reproductive failure and recurrent miscarriage</em>. We will use single cell strand sequencing to map balanced rearrangements and whole genome sequencing of euploid miscarried embryos to identify causative variants.
In <b>CANDLE</b> we want to uncover gene expression patterns associated with <em>APOL1</em> risk alleles and preeclampsia in African American women. We will examine the role of ancestry in mediating the relationship between <em>APOL1</em> genotype and preeclampsia risk. The results can provide insights into genetic and molecular basis of preeclampsia.</p>

<p>This project is in collaboration with</p>
<ul>
  <li><a href="https://scholar.google.at/citations?user=ceRslzAAAAAJ&amp;hl=en">Francesca Antonacci</a>, University of Bari Aldo Moro</li>
  <li><a href="https://scholar.google.at/citations?user=02eKUFwAAAAJ&amp;hl=en&amp;oi=ao">Carlo Alviggi</a>, University of Naples Federico II</li>
  <li><a href="https://scholar.google.at/citations?user=iukICNwAAAAJ&amp;hl=en&amp;oi=ao">Antonio Lamarca</a>, University of Modena and Reggio Emilia</li>
  <li><a href="">Marcella Vacca</a>, National Research Council</li>
  <li><a href="https://scholar.google.at/citations?user=L1PKDnYAAAAJ&amp;hl=en&amp;oi=ao">Gabriella Lania</a>, National Research Council</li>
  <li><a href="https://www.merigen.it/">Imma and Sebastiano Di Biase</a>, MeriGen Research, Italy</li>
  <li>Antonio Capalbo, Juno Genomics</li>
  <li><a href="http://docente.unife.it/michele.rubini">Michele Rubini</a> University of Ferrara</li>
  <li><a href="https://www.monash.edu.my/science/staff/academic/qasim-ayub">Qasim Ayub</a>, Monash University, Malaysia</li>
</ul>

<h4 id="funding"><em>Funding</em></h4>
<ul>
  <li>PRIN 2020J84FAM Ministero dell’Universita e della Ricerca</li>
  <li>PRIN 2022WEHP7M Ministero dell’Universita e della Ricerca</li>
  <li>PRIN P2022ZE75A Ministero dell’Universita e della Ricerca</li>
  <li>Merigen Research s.r.l</li>
  <li>POR Campania FSE 2014-2020 ASSE III – Ob. Sp. 14</li>
</ul>

<p><img src="images/loghi_grep.png" alt="loghi_grep" width="30%" />
<img src="images/miur.jpg" alt="miur" width="30%" /></p>

<h2 id="admixture-mapping-and-pangenomics-analysis-in-the-biorepository-and-integrative-genomics-big-initiative-cohort">Admixture Mapping and Pangenomics Analysis in the Biorepository and Integrative Genomics (BIG) Initiative Cohort</h2>

<p><i>Improving Phenotype-Genotype Linkages through Ancestry Patterns</i></p>

<p>The BIG initiative is currently recruiting 30k participants from Memphis (TN), mostly children, with plans to include a total of 100,000 samples over the next five years. Nearly 45% of current participants are of African ancestry, and the cohort is partnering with the Genomic Information Commons, a consortium of top children’s hospitals, to conduct genomics research aimed at discovering the genetic foundations of human disease in diverse populations.</p>

<p>We want to understand the extent and origin of admixture patterns in the Biorepository and Integrative Genomics (BIG) Initiative cohort and to develop pipelines for admixture mapping of phenotypic traits related to diseases and natural selection phenotypes.</p>

<p><img src="images/panmix.png" alt="panmix" width="30%" /></p>

<p>We are using existing exome sequence data and electronic health records from 10k children to obtain an improved set of genetic variants, including structural variants through mapping against reference pangenomes and imputation.</p>

<p>We will reconstruct the demographic history of the individuals in the cohort at individual and population levels, determine the identity of the ancestral populations and deconvolute their contribution to the genome of single individuals. The ancestry information will be used to identify patterns of natural selection and map phenotypic traits.</p>

<p>This project is in collaboration with</p>
<ul>
  <li><a href="https://scholar.google.com/citations?user=OYJMYwIAAAAJ&amp;hl=en">Robert Williams</a>, UTHSC</li>
  <li><a href="https://scholar.google.com/citations?user=965VipkAAAAJ">Robert Davis </a>, UTHSC</li>
  <li><a href="https://scholar.google.com/citations?user=DuzpZDEAAAAJ">Akram Mohammed</a>, UTHSC</li>
</ul>

<p><img src="images/UTHSC-primary-stacked-logo-4c.jpg" alt="uthsclogo" width="30%" /></p>

<h2 id="mouse-pangenomics">Mouse Pangenomics</h2>

<p>Mice members of BXDs family have been inbred for 20-200 generations. They are of great value for mapping complex traits and phenome-wide association analyses. Current genomic studies on BXD assume a single linear reference genome, making it difficult to observe sequences diverging from the reference, therefore limiting the accuracy and completeness of analyses. Pangenome models overcome this limitation as they contain the full genomic information of a species.</p>

<p>We are building a reference pangenome for all extant members of all BXD families leveraging third generation and 10X sequence data. We will analyze the genetic variation in relation to thousands of phenotypes in the <a href="https://genenetwork.org/">GeneNetwork</a> database.</p>

<div style="border: 2px solid gray; display: inline-block;">
<figure>
  <img src="images/pangenome.png" alt="pangenomeMouse" width="80%" />
  <figcaption><b>(A)</b> odgi-vizlinear visualization of the pangenome of chromosome 19. Each line represents a haplotype. Line interruptions (white) are insertions in one or more strains, therefore deletions in the others (vertical white stripes). The left side is the centromere, the right side is the telomere.In these two regions sequences are fragmented. <b>(B)</b> Extract of the pangenome from the Zfp91gene showing a 2,006 bp insertion found in DBA/2J and 48% of the BXD strains(green nodes in the graph). The insertion is in complete linkage with two other insertions of 4 bp and 135 bp in a region spanning 2.8 kbp. <b>(C)</b> Strain-specific haplotypes (gray segments are not in scale)</figcaption>
</figure> 
</div>

<p>This project is in collaboration with the Department of Genetics, Genomics and Informatics, University of Tennessee Health Science Center, Memphis, TN:</p>
<ul>
  <li><a href="https://scholar.google.com/citations?user=OYJMYwIAAAAJ&amp;hl=en">Robert Williams</a></li>
  <li><a href="https://davidashbrook.wordpress.com/">David Ashbrook</a></li>
  <li><a href="https://thebird.nl/">Pjotr Prins</a></li>
  <li><a href="http://www.hypervolu.me/~erik/erik_garrison.html">Erik Garrison</a></li>
</ul>

<p><em>Our biorxiv preprints on this project can be find <a href="https://www.biorxiv.org/content/10.1101/2022.04.21.489063v1.full">here</a> and <a href="https://www.biorxiv.org/content/10.1101/2022.03.02.482700v1.full[">here</a></em></p>

<p><img src="images/UTHSC-primary-stacked-logo-4c.jpg" alt="uthsclogo" width="30%" /></p>

<p>[== <em>Training in Bioinformatics</em>
We are actively organizing and participating into bioinformatics training
== <em>Science communication</em>]</p>

<h1 id="people"><em>People</em></h1>

<hr />

<p>|Vinc(Enza) Colonna | <img src="images/vcolonna.jpg" alt="Vc" width="40%" /> |I am a genomicist and an expert in human evolutionary and population genomics and bioinformatics.|
|—|—|—|</p>

<p><em>I graduated in Evolutionary Biology from the University of Naples Federico II and did postdoctoral research at the University of Ferrara (Italy) and at Wellcome Trust Sanger Institute in Cambridge (UK). I am now leading the Population genomics laboratory  at the IGB-CNR (Naples, Italy) and I am visiting Assistant Professor at the University of Tennessee, College of Medicine, in the Department of Genetics, Genomics and Informatics</em></p>

<p>In my postdoctoral research I was part of the international consortium 1000 Genomes[PMID: 26432245; 23128226] where I led contributions to two specific aspects. First, I contributed to develop FunSeq [PMID: 24092746], a tool that integrates non-coding information from relevant biological databases for the functional characterization of non-coding variants. Second, I lead a genome-wide scan to identify genomic regions with exceptionally high levels of population differentiation [PMID: 24980144] demonstrating that these regions are enriched for positive selection events and that one half may be the result of classic selective sweeps. Findings from both sub-projects have since been applied to demographic inference and the molecular diagnosis of cancer and myeloid malignancies [PMID: 27121471, 22446628], and to deeper studies on positive selection at the ABCA12 gene [PMID: 30890716].</p>

<p>During my PhD I worked on human isolated populations contributing to characterize several isolated populations, describing the genomic consequences of isolation [PMID: 17476112, 19550436, 22713810], contributing to genetic association studies [PMID: 16611673, 18162505] and to characterize rare variation [PMID: 28643794]</p>

<h5 id="see-my-full-cv-here">See my full C.V. <a href="https://github.com/ColonnaLab/laboratory_WebPage/tree/master/docs/CV_EnzaColonna.pdf">here</a></h5>

<h5 id="link-to-my-google-scholar">Link to <a href="https://scholar.google.com/citations?user=ufP1EYgAAAAJ&amp;hl=en&amp;oi=ao">my Google Scholar</a></h5>

<p>I founded and led <a href="http://www.igb.cnr.it/obilab">OBiLab</a>, a project on training in Bioinformatics</p>

<hr />

<p>|Silvia Buonaiuto | <img src="images/silvia.jpg" alt="silvia" width="40%" /> |I am a postdoc … .|
|—|—|—|</p>

<p><em>I have a PhD degree From the university Luigi Vanvitelli, a master’s degree in Biology from the University of Napoli Federico II. I did a master thesis in molecular biology at the Department of Biology.</em></p>

<p>I work on project DELIVER Decipher unExpLored genetIc Variation inrEproductive failuRe*. My project studies idiopathic recurrent miscarriage to identify genetic variants likely to be causative and ultimately improve prenatal diagnosis.</p>

<hr />

<p>|Franco Marsico | <img src="images/franco.jpeg" alt="franco" width="50%" /> |I am a postdoc with expertise in mathematical modeling applied to genetics.|
|—|—|—|</p>

<p><em>I earned my degree in Biology from the University of Buenos Aires, Argentina, where I also completed my PhD in Computational Biology at the Calculus Institute. My research primarily focused on developing mathematical models for kinship inference, employing a Bayesian Approach. I am a postdoc in the Colonna lab, where my work centers on population genomics.</em></p>

<p>I am currently working on “Admixture Mapping and Pangenomics Analysis in the Biorepository and Integrative Genomics (BIG) Initiative Cohort project”. My focus is on studying recent natural selection signals in admixed populations. Additionally, I have a deep interest in evolution and how to compute processes that shape the history of life.</p>

<h5 id="link-to-my-github-profile">Link to my <a href="https://github.com/MarsicoFL">Github profile</a></h5>

<h5 id="link-to-my-google-scholar-1">Link to <a href="https://scholar.google.ca/citations?user=Vd4yh9wAAAAJ&amp;hl=en">my Google Scholar</a></h5>

<hr />

<p>|Gianluca Damaggio | <img src="images/gianluca.jpg" alt="GL" width="40%" /> |I am a PhD student |
|—|—|—|</p>

<p><em>I am a PhD student at the University of Naples Federico II,  visiting student at the IGB-CNR in Naples, and a Junior Research Fellow at the University of Milano Statale in the Laboratory of Elena Cattaneo.</em></p>

<p>my projects my interests Project Title: HD-DittoGraph - a digital human Embryonic Stem Cell platform for Hungtinton’s repeats*. My project aims at acquiring the ability to  precisely detect perturbations of short tandem repeats of the Huntington’s gene in proliferative cells, leveraging third-generation sequencing data. Currently,</p>

<hr />

<p>|Flavia Villani | <img src="images/Flavia.JPG" alt="flavia" width="40%" /> |I am a Ph.D. student with expertise in pangenomics and comparative genomics|
|—|—|—|</p>

<p><em>I obtained a Master’s degree in Medical Biotechnology from the University of Naples Federico II in Italy. Currently, I am a PhD candidate at the University of Tennessee Health Science Center in the Colonna lab.</em></p>

<p>I am working on the “Mouse Pangenomics Project”. My research aims to construct the pangenome graph of model organisms, specifically inbred mice and rats, using a combination of short and long-read sequence data. This will enable genome-wide association studies to be performed directly on the pangenome. Additionally, I have a strong interest in understanding how mobile genetic elements have driven genome evolution through various mechanisms.</p>

<h5 id="link-to-my-github-profile-1">Link to my <a href="https://github.com/Flavia95">Github profile</a></h5>

<h5 id="link-to-my-google-scholar-2">Link to my <a href="https://scholar.google.com/citations?user=CrklVWcAAAAJ&amp;hl=en">Google Scholar</a></h5>

<hr />

<p>|Rosanna Maione | <img src="images/flavia.jpg" alt="flavia" width="40%" /> |I am a Research Associate. I am an expert in human embriology |
|—|—|—|</p>

<h3 id="former-members">Former members</h3>
<ul>
  <li>Madeleine Emms, Postdoctoral fellow, 2022-2023</li>
  <li>Marialaura Zitiello, Master Student, 2022-2023</li>
  <li>Antonella Mecca, Master Student, 2022-2023</li>
  <li>Angela Sequino, Master Student, 2022-2023</li>
  <li>Davide D’angelo, Visiting master student, 2022</li>
  <li>Giuliana D’Angelo, Master Student, 2019-2020</li>
  <li>Roberto Sirica, PhD student, 2015-2018</li>
  <li>Gaia Leandra Cecere, undergraduate student, 2018</li>
  <li>Marianna Buonaiuto, visiting Postdoc, 2017</li>
  <li>Lucia De Martino, visiting master Student, 2016</li>
</ul>

<h2 id="publications"><em>Publications</em></h2>

<p><a href="https://scholar.google.it/citations?user=ufP1EYgAAAAJ&amp;hl=en&amp;oi=ao">Google Scholar</a></p>

<h2 id="past-research"><em>Past Research</em></h2>

<p>At this [link] (https://github.com/ColonnaLab/laboratory_WebPage/blob/master/docs/pastResearc.adoc[ is possible to found our past Research)</p>

<h3 id="contacts">Contacts*</h3>

<p>Vincenza Colonna, PhD</p>

<p>Istituto di Genetica e Biofisica “Adriano Buzzati-Traverso”
piano R, stanza 11
via Pietro Castellino 111 - 80131 Napoli - Italy - <a href="https://www.google.co.uk/maps/place/Cnr+Consiglio+Nazionale+Delle+Ricerche/@40.8545777,14.2250388,17z/data=!4m6!1m3!3m2!1s0x133b08ed1ffff6c1:0xede26e8ccb5ccb3b!2sCnr+Consiglio+Nazionale+Delle+Ricerche!3m1!1s0x133b08ed1ffff6c1:0xede26e8ccb5ccb3b">map</a>
tel. +39 081 6132 254</p>

<p>Email: enza.colonna@gmail.com  - vincenza.colonna@igb.cnr.it</p>
