= *Past Research Interests*

== *GReP* - Genomics of Recurrent Pregnancy Loss

Approximately 70% of all human fertilizations are naturally aborted, with most occurring before the mother knows she is pregnant. Miscarriages that occur later in pregnancy are often due to non-viable genetic variation and this is very likely in the case of Recurrent Pregnancy Losses (RPL).

Numerical and structural chromosomal abnormalities are routinely investigated to understand causes of RPL, while the study of small-size sequence variants that can impact gene function has been neglected, leaving a whole range of possible causes unexplored.

With this project we want to identify variants severely disrupting genes and regulatory functions, therefore incompatible with life, using whole genome sequence of DNA extracted from chorionic villi of miscarriages collected from women with RPL.

*GReP code* is shared https://github.com/ezcn/grep[here]

{empty} +
{empty} +
{empty} +
{empty} +

== *HD-DittoGraph* - a digital human Embryonic Stem Cell platform for Hungtinton’s repeats

This project aims to identify the genetic factors that are implicated in instability of the CAG repeats in the Hungtinton's gene _HTT_, both during mitotic cell replication and in post-mitotic neurons. We use a human embryonic stem (hES) cellC-based cell platform, barcoded DNA libraries, CRISPR technologies and long-read third generation DNA sequencing.
//Factors modulating CAG elongation have been divided into cis-acting elements (i.e. DNA sequences in the proximity of the repeat or the repeat itself) and trans-acting elements (i. e. other proteins) whose interaction with the repeat contributes to its instability.

[#img-count]
[caption="Figure 1.2: ",link=https://github.com/GianlucaDamaggio/testWebPage/blob/master/image/strique_count.png]
image::/images/strique_count.png[preSeq_screening,800,600,role="center"]

CAG sizing using STRIque on Nanopore sequencing data obtained from libraries generated starting from a donor plasmid carrying the sequence [CAG]48-CAA-CAG. Several combinations of parameters (prefix/suffix size, Qscore, lenght of the reference sequence) were used. Accurate CAG size was detected under most of the combinations used.


=== *Collaborations*
[none]
- http://www.cattaneolab.it/?page_id=42&lang=en[Elena Cattaneo] - University of Milano, Italy
- https://air.unimi.it/cris/rp/rp11682#.YEDR4C1Q2p0[Dario Besusso] - University of Milano, Italy
- https://elixir-iib-training.github.io/website/instructors/allegra_via.html[Allegra Via] - ELIXIR-IIB Training Platform
- https://www.recas-bari.it/index.php/it/[ReCaS] - data center
{empty} +
{empty} +
{empty} +
{empty} +

== *SARS-CoV-2* - Genomics of COVID19

With the emerging of the COVID19+ pandemic we feel motivated to give our contribution as geneticists. Among the many aspects worth working on, we chose to deal with the paucity of SARS-CoV-2 genomic sequences from Italy publicly available.


[#img-wolrd]
[caption="Figure 1.2: ",link=https://github.com/GianlucaDamaggio/testWebPage/blob/master/image/wolrd_covid19.png]
image::/images/wolrd_covid19.png[wolrd_covid19,700,align="center"]

Despite Italy being one of the earliest countries hit by the pandemic, as we write (04/05/20) there are only seven sequences from samples collected in Italy in GenBank.


We are committed to collect, sequence and openly publish as many samples as we can. Knowledge of the SARS-CoV-2 genomic sequence will enable basic research on COVID19, including monitoring the pandemic diffusion, improving genetic testing and develop effective vaccines.

*These consideration were presented at the Commissione sanità of the Italian Senate of the Republic in May 2020. Slides can be find https://github.com/ColonnaLab/laboratory_WebPage/blob/master/docs/2020_Covid19Senato.pdf[here]*

//.[purple]#A de novo assembly of pulled down RNA sequenced on a nanopore device.#
//image::images/Assembly_denovo.png[align="center"]

We are working on SARS-CoV-2 genomics along two research lines:

. In collaboration with CEINGE we are sequencing SARS-CoV-2 genomes using third-generation sequencing.

.  We are part of a network that aim to develop a platform for sharing and analyzing genomic sequence information from samples collected in the Region Campania in Italy. This research is supported by Regione Campania – Recover – Covid 19 -DPGR n. 45 (06/03/2020). Prot. 2020, 0213629 del 05/05/2020.

//.  We are recruiting COVID19+ individuals from Regione Basilicata, one of the few regions in Italy with the lowest rate of infection.

//image::images/bloccscheme.png[blocc,800,align="center"]

'''

{empty} +
{empty} +
{empty} +
{empty} +

== Genetics of fruit shape and size in peppers
Capsicum fruits are among the most-widely consumed vegetable crops in the world, and one of the most extensively domesticated plants. While a focus of breeding programs and widely studied for its beneficial properties, the genetics of pepper has been studied mostly by linkage analysis and with the use of coarse sets of markers and low resolution phenotypic data.

We discovered 746k high quality polymorphic sites analyzing sequence data of 373 pepper accessions. We also measured thirty-eight fruit shape and size attributes in 220 C. annuum accessions.We used these data to:

(i) uncover genomic properties of the pepper genome

(ii) describe population structure within the Capsicum genus at a resolution never achieved before

(iii) study natural selection,

(iv) discover significant association between genetic markers and traits related to pepper fruit shape and size in C. annuum, including a non-synonymous change in the sequence of longifolia 1-like gene associated with variance in fruit elongation.

[#img-capsicum]
[caption="Figure 1.2: ",link=https://github.com/GianlucaDamaggio/testWebPage/blob/master/image/preSeq_screaning.png]
image::/images/capsicum.png[capsicum,1000]

Our collection includes 373 pepper accessions from eleven species from 51 countries. It covers all the economically important species of Capsicum widely used in breeding programs, representing the largest study so far in terms of number of species and number of genetic variants analyzed.

This project is a collaboration with Pasquale Tripodi and Nunzio D'Agostino at http://sito.entecra.it/portale/cra_dati_istituto.php?id=207[CREA]

>> See our open access https://www.nature.com/articles/s41598-019-46136-5[paper]!! <<

>> Go to the project https://ezcn.github.io/Capsicum-genomics/[website]! <<

{empty} +
{empty} +
{empty} +
{empty} +

== Positive selction at the ABCA12 gene

Natural selection acts on genetic variants by increasing the frequency of alleles responsible for a cellular function that is favorable in a certain environment.

In a https://genomebiology.biomedcentral.com/articles/10.1186/gb-2014-15-6-r88[previous genome-wide scan for positive selection] in contemporary humans, we identified a signal of positive selection in European and Asians at the genetic variant rs10180970. The variant is located in the second intron of the http://www.ensembl.org/Homo_sapiens/Gene/Summary?db=core;g=ENSG00000144452;r=2:214931542-215138428[ABCA12] gene, which is implicated in the lipid barrier formation and down-regulated by UVB radiation. Mutations in ABCA12 causes https://omim.org/entry/242500[Harlequin Ichthyosis], a rare recessive disorderd with variable penetrance.

We studied the signal of selection in the genomic region surrounding rs10180970 in a larger dataset that includes DNA sequences from ancient samples. We also investigated the functional consequences of gene expression of the alleles of rs10180970 and another genetic variant in its proximity in healthy volunteers exposed to similar UV radiation.

image::/images/abca12.png[abca12,800]

The Cross Population Extended Haplotype Homozogysity statistic (XP-EHH), measured between pairs of continental populations, shows a signal of positive selection in non-Africans over 35kb downstream rs10180970, especially in East-Asians.


>> See our open access https://www.nature.com/articles/s41598-019-40360-9[paper]!! <<

{empty} +
{empty} +
{empty} +
{empty} +

== Deleterious mutations in human consanguineous populations

Alleles with detrimental consequences and recessive inheritance pattern are the major contributors to the genetic load in human populations. Indeed, while dominant deleterious variants are quickly purged by selection, recessive variants remain in the populations at low frequency and mostly in a heterozygosis.

image:/images/consang_pop.png[consang_pop,600]

In consanguineous populations kin marriages can cause deleterious recessive variants to be exposed in homozygous state causing higher rate rate of disorders (congenital, late onset, infertility, miscarriage, infant mortality and morbidity). At the same time, however, inbreeding provides an occasion for selection to act on deleterious phenotypes, in a phenomenon known as purging through inbreeding.

We are interested in understanding and characterizing the incidence of deleterious mutations in consanguineous populations.

{empty} +
{empty} +
{empty} +
{empty} +
